date_range/models/date_range_search_mixin.py

```diff
@@ -97,6 +97,8 @@ def get_views(self, views, options=None):
         fields list (while still showing up in the Export widget)
         """
         result = super().get_views(views, options=options)
-        if "date_range_search_id" in result["models"][self._name]:
-            result["models"][self._name]["date_range_search_id"]["string"] = _("Period")
+        if "date_range_search_id" in result["models"][self._name]["fields"]:
+            result["models"][self._name]["fields"]["date_range_search_id"]["string"] = (
+                _("Period")
```

Comment:
FYI, Since odoo 18.0, **`_`** should now be called from the env to save time: **`self.env._()`**. See: [Link Here](https://github.com/odoo/odoo/commit/b794f0f332f473deb2c04eba60baf4761db3b508)
```suggestion
self.env._("Period")
```